cmd/evm/runner.go

```diff
@@ -274,24 +274,28 @@ func runCmd(ctx *cli.Context) error {
 	if ctx.Bool(CreateFlag.Name) {
 		input = append(code, input...)
 		execFunc = func() ([]byte, uint64, error) {
+			// don't mutate the state!
+			runtimeConfig.State = prestate.Copy()
 			output, _, gasLeft, err := runtime.Create(input, &runtimeConfig)
 			return output, gasLeft, err
 		}
 	} else {
 		if len(code) > 0 {
-			statedb.SetCode(receiver, code)
+			prestate.SetCode(receiver, code)
 		}
 		execFunc = func() ([]byte, uint64, error) {
+			// don't mutate the state!
+			runtimeConfig.State = prestate.Copy()
 			output, gasLeft, err := runtime.Call(receiver, input, &runtimeConfig)
 			return output, initialGas - gasLeft, err
 		}
 	}
 
 	bench := ctx.Bool(BenchFlag.Name)
-	output, stats, err := timedExec(bench, execFunc)
+	output, stats, execErr := timedExec(bench, execFunc)
```

Comment:
```suggestion
	output, stats, err := timedExec(bench, execFunc)
```

Reply:
The naming is a carryover from when I added a separate error return from `timedExec` to differentiate between "execution error within the EVM" and "catastrophic error as a result of the implementation being faulty".  Now we only have the former, so renaming to `err` is fine.